### PR TITLE
Update readme to include installation

### PR DESCRIPTION
**Pull request by JakeSidSmith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
...
+
+1. Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+1. Using bower
+  ```
+  bower install react-reorderable
```
Comment (adsteel) [inline README.md]: react-reorder?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
...
+
+* Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+* Using bower
+  ```
+  bower install react-reorderable
```
Comment (adsteel) [inline README.md]: react-reorder? says react-reorderable

File: README.md (hunk 1 above)
Comment (JakeSidSmith) [inline README.md]: I published to bower a while ago, and managed to get `react-reorderable`, but I don't think I should update now as people may already be using it.

File: README.md (hunk 2 above)
Comment (JakeSidSmith) [inline README.md]: I published to bower a while ago and manage to get `react-reorderable` as the name, but I don't think I should update it now as people may already be using it.

Comment (adsteel) [inline README.md]: Do you think it's worth aligning on bower? Seems like it may cause confusion down the line to have two names.

Comment (adsteel) [inline README.md]: As in, leave the old name on bower, copy the repo to the new name, add the new name to bower, update NPM and the old repo to point to the new repo?

Comment (JakeSidSmith) [inline README.md]: I could unregister it from bower & register under another name... I just think I may cause a lot of people problems. :/

Comment (adsteel) [inline README.md]: Hm, that wouldn't be good. I just wonder if having two names won't be confusing to people. Maybe there's a way to deprecate the old name? I've been considering a name change for one of my libraries, too, and I'm not sure what the best way is to go about it.

Comment (JakeSidSmith) [inline README.md]: Ok, it seems I can have both names & keep it linked. XD

I've just registered under `react-reorder` also. Will update the readme to use that.

May have to remove the old package from bower at some point.

**Comment by JakeSidSmith** [comment]
Closed for name update #53

**Pull request closed without merging**